docs/tutorial/gatsby-image-tutorial

```diff
@@ -0,0 +1,275 @@
+
+The first thing to know is that an alias is assigning a name to a query. Below is a simple example.
+
+```jsx
+talks: allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+When you do that, you’ve changed the reference to that object in your JSX. While it was previously referenced as
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
+```
+
+giving it an alias does not add a level of complexity to the response object, it just replaces it. So you end up with the same structure referenced as
+
+```jsx
+{data.talks.edges.map(({ node }) => ())
+```
+
+The top-level object name of data is implicit. This is important because when you conduct multiple queries as part of a single component `Gatsby` still passes the entire result to the component.
+
+```jsx
+const SpeakingPage = ({ data}) => {})
+```
+
+everything else gets referenced from that top-level return
+name.
+
+With that understanding, you can combine two queries and use aliasing to distinguish between them.
+
+```jsx
+{
+    allSpeakingYaml {
+        edges {
+            node {
+                image {
+                    childImageSharp {
+                        fluid {
+                            ...GatsbyImageSharpFluid
+                        }
+                    }
+                }
+            }
+        }
+    }
+    banner: file(relativePath: { eq: "headers/default.jpg" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+  }
+}
+```
+
+Notice that this example uses aliasing for one query and not the other. This is allowed; there is no requirement that all your queries use aliasing. In this case, the JSX would look like this to access the `speaking.yaml` content.
+
+```jsx
+{data.allSpeakingYaml.edges.map(({ node }) => ())
+```
+
+And then like this to access the image using the alias name `banner`.
+
+```jsx
+<Img fluid={data.banner.childImageSharp.fluid} />
+```
+
+### Aspect Ratio
+
+These examples should handle a fair number of your use cases. A couple bonus things. First, `gatsby-image` has a nice feature that gives you the ability to set an aspect ratio. This can be used for fixed or fluid processed images, it doesn't matter.
+
+```jsx
+<Img sizes={{ ...data.banner.childImageSharp.fluid, aspectRatio: 21 / 9 }} />
+```
+
+### Bonus Error
+
+Now for errors to watch out for. If you change your image processing from `fixed` to `fluid` you may see this error.
+
+![In image cache error message.](https://tenmilesquare.com/wp-content/uploads/2019/02/Screen-Shot-2019-02-18-at-11.46.03-AM-1024x212.png)
```

Comment:
We should import the image here in case that URL changes! Can you save the file locally and add it to this PR? You'd want to create a directory called `gatsby-image-tutorial` and then you can save the index.md and image in the same place.